Assets/Scripts/LetterBlock.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using UnityEngine;
+
+/// <summary>
+/// Choose letter block color, handle shake and break animation
+/// </summary>
+public class LetterBlock : MonoBehaviour
```

Comment:
nit: `[RequireComponent(typeof(SpriteRenderer))]` since you call `GetComponent` and don't check the returned value for null